### DISCUSSION TITLE
Web Installer ...

### DISCUSSION THREAD

**Discussion by helenfromyorkshire** [PlummersSoftwareLLC/NightDriverStrip#315] [General]
Hi,

For the M5Stck Cplus 1.1 ... what data pin is used?

Thanks.

**Comment by davepl** [comment]
You can change it in the code as LED0, but by default it should be pin 26, which is on the grove port on the end along with power and ground

**Comment by helenfromyorkshire** [comment]
Thank you for your help Mr Plummer.

On my M5StickCPlus the pins are:

Grove: GND, V-out, G32,G33
PINS: GND, 5V-out,G26, G36/G25, G0, BAT,3V3,5V-in

I have my LEDS linked to the grove plug ... just GND and G32.
Unfortunately nothing happens when the code is iploade by the web installer.

Thanks again.
Helen

**Comment by davepl** [comment]
I assume you’re powering the M% via USB?  I forget which is which, but one uses pin 26 and one uses pin 32.  Try the other in the code, see if it works with changing 26 to 32 or vice versa!
